lib/api/WP_Auth0_Api_Delete_User_Mfa.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Contains WP_Auth0_Api_Delete_User_Mfa.
+ *
+ * @package WP-Auth0
+ *
+ * @since 3.8.0
+ */
+
+/**
+ * Class WP_Auth0_Api_Delete_User_Mfa to perform a client credentials grant.
+ */
+class WP_Auth0_Api_Delete_User_Mfa extends WP_Auth0_Api_Abstract {
+
+	/**
+	 * Default value to return on failure.
+	 *
+	 * @var integer
+	 */
+	const RETURN_ON_FAILURE = 0;
+
+	/**
+	 * Decoded token received.
+	 *
+	 * @var null|object
+	 */
+	protected $token_decoded = null;
+
+	/**
+	 * WP_Auth0_Api_Delete_User_Mfa constructor.
+	 *
+	 * @param WP_Auth0_Options                $options - WP_Auth0_Options instance.
+	 * @param WP_Auth0_Api_Client_Credentials $api_client_creds - WP_Auth0_Api_Client_Credentials instance.
+	 */
+	public function __construct(
+		WP_Auth0_Options $options,
+		WP_Auth0_Api_Client_Credentials $api_client_creds
+	) {
+		parent::__construct( $options );
+		$this->api_client_creds = $api_client_creds;
+	}
+
+	/**
+	 * Set the user_id and provider, set the authorization header, call the API, and handle the response.
+	 *
+	 * @param string|null $user_id - Auth0 user ID to delete the MFA provider.
```

Comment:
Should we mention in docheader that the default MFA provider is google-authenticator?

Reply:
@cocojoe - Function params default there and it's mostly an internally-used functionality.